src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -485,7 +485,7 @@
        *
        {@comments(util.getDocLines(context.paramComments(service, method)))}
        * @@return {Stream}
-       *   An object stream which emits {@context.typeDocument(methodConfig.getPageStreaming.getResourcesField.getType)} on 'data' event.
+       *   An object stream which emits {@apiMethodView.doc.pageStreamingResourceTypeName} on 'data' event.
```

Comment:
Page streaming rpcs produce two methods in node.

1) the method itself
2) a method that calls 1) and wraps it in a stream.

This part of the snip file pertains to method 2. Method 2 has an entirely different return object than method 1. By moving this return line to the getDocLines function, it will in turn change 1.